src/clincoded/schemas/annotation.json

```diff
@@ -91,6 +91,47 @@
             "title": "Affiliation",
             "description": "The affiliation this annotation is associated with.",
             "type": "string"
+        },
+        "articleNotes": {
```

Comment:
Do we have to update schema version here? Saw that this schema is liked to `curatorHistory`, not sure if we forgot or if it's not necessary due to the link.

Reply:
I think you're right, the version should be updated. I feel like I purposefully left it unchanged for a reason, but I may have just forgotten. I'll add it in.